builtin/providers/aws/provider.go

```diff
@@ -111,121 +111,122 @@ func Provider() terraform.ResourceProvider {
-			"aws_route53_record":                   resourceAwsRoute53Record(),
-			"aws_route53_zone_association":         resourceAwsRoute53ZoneAssociation(),
-			"aws_route53_zone":                     resourceAwsRoute53Zone(),
-			"aws_route53_health_check":             resourceAwsRoute53HealthCheck(),
-			"aws_route":                            resourceAwsRoute(),
-			"aws_route_table":                      resourceAwsRouteTable(),
-			"aws_route_table_association":          resourceAwsRouteTableAssociation(),
-			"aws_s3_bucket":                        resourceAwsS3Bucket(),
-			"aws_s3_bucket_object":                 resourceAwsS3BucketObject(),
-			"aws_security_group":                   resourceAwsSecurityGroup(),
-			"aws_security_group_rule":              resourceAwsSecurityGroupRule(),
-			"aws_spot_instance_request":            resourceAwsSpotInstanceRequest(),
-			"aws_sqs_queue":                        resourceAwsSqsQueue(),
-			"aws_sns_topic":                        resourceAwsSnsTopic(),
-			"aws_sns_topic_subscription":           resourceAwsSnsTopicSubscription(),
-			"aws_subnet":                           resourceAwsSubnet(),
-			"aws_volume_attachment":                resourceAwsVolumeAttachment(),
-			"aws_vpc_dhcp_options_association":     resourceAwsVpcDhcpOptionsAssociation(),
-			"aws_vpc_dhcp_options":                 resourceAwsVpcDhcpOptions(),
-			"aws_vpc_peering_connection":           resourceAwsVpcPeeringConnection(),
-			"aws_vpc":                              resourceAwsVpc(),
-			"aws_vpc_endpoint":                     resourceAwsVpcEndpoint(),
-			"aws_vpn_connection":                   resourceAwsVpnConnection(),
-			"aws_vpn_connection_route":             resourceAwsVpnConnectionRoute(),
-			"aws_vpn_gateway":                      resourceAwsVpnGateway(),
+			"aws_ami":                                 resourceAwsAmi(),
+			"aws_ami_copy":                            resourceAwsAmiCopy(),
+			"aws_ami_from_instance":                   resourceAwsAmiFromInstance(),
+			"aws_app_cookie_stickiness_policy":        resourceAwsAppCookieStickinessPolicy(),
+			"aws_autoscaling_group":                   resourceAwsAutoscalingGroup(),
+			"aws_autoscaling_notification":            resourceAwsAutoscalingNotification(),
+			"aws_autoscaling_policy":                  resourceAwsAutoscalingPolicy(),
+			"aws_autoscaling_schedule":                resourceAwsAutoscalingSchedule(),
+			"aws_cloudformation_stack":                resourceAwsCloudFormationStack(),
+			"aws_cloudtrail":                          resourceAwsCloudTrail(),
+			"aws_cloudwatch_event_rule":               resourceAwsCloudWatchEventRule(),
+			"aws_cloudwatch_event_target":             resourceAwsCloudWatchEventTarget(),
+			"aws_cloudwatch_log_group":                resourceAwsCloudWatchLogGroup(),
+			"aws_cloudwatch_logs_subscription_filter": resourceAwsCloudwatchLogsSubscriptionFilter(),
```

Comment:
Shall we change this to match the existing resource name above - i.e. `aws_cloudwatch_log_subscription_filter`?